<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvUpUpdatePackageWidget</class>
 <widget class="QWidget" name="C_SyvUpUpdatePackageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1,0">
     <property name="spacing">
      <number>19</number>
     </property>
     <property name="leftMargin">
      <number>18</number>
     </property>
     <property name="rightMargin">
      <number>21</number>
     </property>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabHeadingWidget" name="pc_LabelTitle">
       <property name="text">
        <string>UPDATE PACKAGE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabSvgOnly" name="pc_LabelStateIcon">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeLabStateInfo" name="pc_LabelState">
       <property name="text">
        <string>Warnings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>14</width>
         <height>34</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubIconText" name="pc_PushButtonClearAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/system_views/IconClearAllEnabled.svg</normalon>
           <disabledoff>:/images/system_views/IconClearAllDisabled.svg</disabledoff>
           <disabledon>:/images/system_views/IconClearAllDisabled.svg</disabledon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubIconText" name="pc_PushButtonImport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/system_views/IconImportEnabled.svg</normalon>
           <disabledoff>:/images/system_views/IconImportDisabled.svg</disabledoff>
           <disabledon>:/images/system_views/IconImportDisabled.svg</disabledon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubIconText" name="pc_PushButtonExport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/system_views/IconExportEnabled.svg</normalon>
           <disabledoff>:/images/system_views/IconExportDisabled.svg</disabledoff>
           <disabledon>:/images/system_views/IconExportDisabled.svg</disabledon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="stw_opensyde_gui_elements::C_OgePubIconText" name="pc_PushButtonCreatePackage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/images/system_views/IconCreateUpdatePackageEnabled.svg</normalon>
           <disabledoff>:/images/system_views/IconCreateUpdatePackageDisabled.svg</disabledoff>
           <disabledon>:/images/system_views/IconCreateUpdatePackageDisabled.svg</disabledon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="pc_HorizontalListLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgePubIconOnly" name="pc_PushButtonScrollLeft">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>9</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../application.qrc">
            <normaloff>:/images/IconScrollLeftEnabled.svg</normaloff>
            <disabledoff>:/images/IconScrollLeftDisabled.svg</disabledoff>
            <disabledon>:/images/IconScrollLeftDisabled.svg</disabledon>:/images/IconScrollLeftEnabled.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="stw_opensyde_gui::C_SyvUpUpdatePackageListWidget" name="pc_ListWidget"/>
        </item>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgePubIconOnly" name="pc_PushButtonScrollRight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>9</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../application.qrc">
            <normaloff>:/images/IconScrollRightEnabled.svg</normaloff>
            <disabledoff>:/images/IconScrollRightDisabled.svg</disabledoff>
            <disabledon>:/images/IconScrollRightDisabled.svg</disabledon>:/images/IconScrollRightEnabled.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pc_HorizontalLayout2" stretch="1,1,1">
        <item>
         <spacer name="pc_HorizontalSpacer4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="stw_opensyde_gui_elements::C_OgeWiBopperle" name="pc_IndexViewWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="pc_HorizontalSpacer3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubIconOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgePubIconText</class>
   <extends>QPushButton</extends>
   <header>C_OgePubIconText.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeWiBopperle</class>
   <extends>QWidget</extends>
   <header>C_OgeWiBopperle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabHeadingWidget</class>
   <extends>QLabel</extends>
   <header>C_OgeLabHeadingWidget.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabStateInfo</class>
   <extends>QLabel</extends>
   <header>C_OgeLabStateInfo.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SyvUpUpdatePackageListWidget</class>
   <extends>QListWidget</extends>
   <header>C_SyvUpUpdatePackageListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabSvgOnly</class>
   <extends>QLabel</extends>
   <header>C_OgeLabSvgOnly.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../application.qrc"/>
 </resources>
 <connections/>
</ui>
